src/main/java/com/rackspace/ceres/app/services/DataWriteService.java

```diff
@@ -138,4 +147,9 @@ private void cleanTags(Map<String, String> tags) {
         .retryWhen(appProperties.getRetryInsertDownsampled().build())
         .checkpoint();
   }
+
+  private Mono<?> storeMetricGroup(Metric metric) {
+    return reactiveRedisTemplate.opsForSet()
+        .add(metric.getTags().get(LABEL_METRIC_GROUP), Arrays.asList(metric.getMetric()));
```

Comment:
I think I missed this before. Are you storing each metric in the redis database?

Reply:
yes, for every metricGroup there would be a list of metricNames stored in Redis.

Reply:
Oh duh. I didn't realize when I left the comment that getMetric was just the metric name.